src/database/DataTypes/Transaction.js

```diff
@@ -597,6 +597,36 @@ export class Transaction extends Realm.Object {
   get isRemoteOrder() {
     return !this.linkedRequisition;
   }
+
+  toJSON() {
+    return {
+      id: this.id,
+      serialNumber: this.serialNumber,
+      otherParty: this.otherParty,
```

Comment:
I've been in general keeping id's of other objects rather than objects. In this case, `otherParty` is a live realm object which shouldn't matter as we don't edit it, but would require editing in a transaction which would be weird when you think it's a plain JS object.

My argument for keeping IDs is
1) immutably updating nested objects is a pain
2) Keeping potentially multiple different versions of an object in memory can cause some issues with objects going stale so easier to just use FK id's

Reply:
thanks. and oops - I missed that one. Have removed, I don't think we'll need `otherParty` right now, and can add it back if needed.